pkgs/build-support/mkshellwrapper/default.nix

```diff
@@ -0,0 +1,94 @@
+{ mkShell
+, writeScript
+, bashInteractive
+, lib
+, writeText
+, coreutils
+}:
+
+# This function creates a wrapper script that runs a command in a shell context.
+{ # The shell that will execute the wrapper script
+  shell ? bashInteractive + "/bin/bash"
```

Comment:
Should this be the interactive shell to begin with?

Reply:
TBH I don't know but it's customizable

Reply:
Changed to bash

No problems so far